docs/cog_guides/trivia.rst

```diff
@@ -0,0 +1,414 @@
+
+.. _trivia-command-triviaset-timelimit:
+
+^^^^^^^^^^^^^^^^^^^
+triviaset timelimit
+^^^^^^^^^^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]triviaset timelimit <seconds>
+
+**Description**
+
+Sets the maximum time permitted to answer a question.
+
+**Arguments**
+
+* ``<seconds>``: The number of seconds to wait for an answer.
+
+.. _trivia-command-triviaset-custom:
+
+^^^^^^^^^^^^^^^^
+triviaset custom
+^^^^^^^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]triviaset custom
+
+**Description**
+
+Manage custom trivia lists. See :ref:`creating custom trivia lists <trivia-list-creation>`
+for more information.
+
+.. _trivia-command-triviaset-custom-upload:
+
+^^^^^^^^^^^^^^^^^^^^^^^
+triviaset custom upload
+^^^^^^^^^^^^^^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]triviaset custom upload
+
+**Description**
+
+Upload a custom trivia list. The bot will prompt you to upload 
+your list as an attachment in Discord.
+
+.. _trivia-command-triviaset-custom-list:
+
+^^^^^^^^^^^^^^^^^^^^^
+triviaset custom list
+^^^^^^^^^^^^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]triviaset custom list
+
+**Description**
+
+List all uploaded custom trivia lists.
+
+.. _trivia-command-triviaset-custom-delete:
+
+^^^^^^^^^^^^^^^^^^^^^^^
+triviaset custom delete
+^^^^^^^^^^^^^^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]triviaset custom delete <name>
+
+**Description**
+
+Delete a custom trivia list.
+
+**Arguments**
+
+* ``<name>``: The name of the custom list to be deleted.
+
+.. _trivia-command-trivia:
+
+^^^^^^
+trivia
+^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]trivia [categories...]
+
+**Description**
+
+Start a trivia session on the specified category.
+
+Multiple categories can be listed, in which case the trivia session 
+will use all of the specified lists to select questions from.
+
+**Arguments**
+
+* ``[categories...]``: The category to play. Can be multiple.
+
+.. _trivia-command-trivia-leaderboard:
+
+^^^^^^^^^^^^^^^^^^
+trivia leaderboard
+^^^^^^^^^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]trivia leaderboard
+
+**Description**
+
+Shows the trivia leaderboard. Defaults to the top ten in the 
+current server, sorted by total wins. The subcommands provide 
+more customized leaderboards.
+
+.. _trivia-command-trivia-leaderboard-global:
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+trivia leaderboard global
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]trivia leaderboard global [sort_by=wins] [top=10]
+
+**Description**
+
+The global trivia leaderboard.
+
+**Arguments**
+
+* ``[sort_by=wins]``: The method by which to sort the leaderboard (defaults to wins). Can be one of: 
+  * ``wins``: total wins
+  * ``avg``: average score
+  * ``total``: total correct answers from all sessions
+  * ``games``: total games played.
+
+* ``[top=10]``: The number of ranks to show on the leaderboard. Defaults to 10
```

Comment:
```suggestion
- ``[sort_by=wins]`` The method by which to sort the leaderboard (defaults to wins). Can be one of:
    - ``wins`` Total wins
    - ``avg`` Average score
    - ``total`` Total correct answers from all sessions
    - ``games`` Total games played.

- ``[top=10]`` The number of ranks to show on the leaderboard. Defaults to 10
```

Reply:
Must be tab indented to format correctly. Otherwise looks like this:
![image](https://user-images.githubusercontent.com/955640/98388409-b99c4280-2020-11eb-8716-d3c357ee93b7.png)